src/main/java/seedu/address/storage/JsonAdaptedTag.java

```diff
@@ -28,6 +29,10 @@ public JsonAdaptedTag(Tag source) {
         tagName = source.tagName;
     }
 
+    /**
+     * Returns the {@code tagName} for Jackson to use.
```

Comment:
There's probably no need to tag tagName with the code formatter since its private anyways

Reply:
hmm good point let me try seeing if it throws anything weird